.github/workflows/deployment.yml

```
@@ -0,0 +1,35 @@
+name: OpenEduHub - Github Page
+
+on:
+  push:
+    branches:
+      - main
```

```suggestion
      - main
      - x86_64
```
Change this workflow to also deploy the `x86_64` branch under the `/hardware-software-interface/x86_64` base URL.